src/js/utils/textFormat.js

```diff
@@ -19,7 +20,7 @@ export function removeTwitterNameFromDescription (displayName, twitterDescriptio
     } else if (twitterDescriptionNotNull.startsWith("The " + displayNameNotNull)) {
       twitterDescriptionMinusName = twitterDescriptionNotNull.substr(displayNameNotNull.length + 4);
     } else if (twitterDescriptionNotNull.length) {
-      twitterDescriptionMinusName = ". " + twitterDescriptionNotNull;
+      twitterDescriptionMinusName = twitterDescriptionNotNull;
```

Comment:
It's clever how the description was made to joined with the org name, but moving forward we'll want these separate. For now, I just removed the period.